### PR TITLE
Refine imports and type annotations

### PR DESCRIPTION
**Pull request by akainth015**
Signed-off-by: Aanand Kainth <aanand@akainth.me>

### CODE REVIEW

File: bentoml/adapters/dataframe_input.py [hunk 1]
```diff
@@ -55,6 +58,92 @@ class DataframeInput(BaseInputAdapter):
         ValueError: Incoming data format can not be handled. Only json and csv
     """
 
+    def from_cli(self, cli_args: Tuple[str, ...]) -> Iterator[InferenceTask]:
+        parser = argparse.ArgumentParser()
```
Comment (bojiang) [inline bentoml/adapters/dataframe_input.py:62]: Why not call `parse_cli_input` here?

Comment (akainth015) [reply bentoml/adapters/dataframe_input.py:62]: I don't think I can add the `json` and `csv` options then, right?

Comment (bojiang) [reply bentoml/adapters/dataframe_input.py:62]: We could get the file ext ".csv" or ".json" like this:

for bin_io in parse_cli_input(cli_args):
    file_ext = bin_io.name

Comment (akainth015) [reply bentoml/adapters/dataframe_input.py:62]: Oh ok, what would I do for the string input?

File: bentoml/adapters/base_input.py [hunk 2]
```diff
@@ -6,17 +6,17 @@
 
 # http://www.apache.org/licenses/LICENSE-2.0
 
+import argparse
+import contextlib
+import functools
 # Unless required by applicable law or agreed to in writing, software
```
Comment (bojiang) [inline bentoml/adapters/base_input.py:12]: ?

Comment (akainth015) [reply bentoml/adapters/base_input.py]: I've got no idea what happened there, I couldn't reproduce it with either `dev/format` nor PyCharm reformatting.

**Comment by bojiang** [comment]
LGTM & Merge

**Pull request merged**